app/components/forms/wizard/basic-details-step.js

```diff
@@ -442,7 +407,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
         event,
         type,
         position,
-        quantity      : 100,
+        quantity      : 100,	
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)